containers/release.Containerfile

```diff
@@ -0,0 +1,78 @@
+FROM registry.access.redhat.com/ubi9-minimal:9.2-691 as builder
+
+ARG WAR_FILE
```

Comment:
A few questions:
- Do we want to build production image from war or from RPM? E.G. built in PNC?
- If we we want war. What's the benefit of getting the war as a param compared to building it in the dockerfile.